cruiser/scenarios/eg01_eg24.json

```diff
@@ -0,0 +1,1125 @@
+{
```

Comment:
Please remove this file from the repository.  JSON simulation files should not be in the repo.